hack/heartbeat_account.py

```diff
@@ -26,9 +26,7 @@
 USER = "cluster-api-provider-aws"
 
 if __name__ == "__main__":
-    count = 0
-    # keep sending heart beat for 3 hours
-    while count < 180:
+    while True:
         conn = httplib.HTTPConnection(BOSKOS_HOST)
         print "POST-ing heartbeat for resource %s to %s" % (BOSKOS_RESOURCE_NAME, BOSKOS_HOST)
         conn.request("POST", "/update?%s" % urllib.urlencode({
```

Comment:
There's another ref to count at the bottom

Reply:
lol, yeah caught that as I was backporting it to the `release-0.4` branch.